pkg/scalers/postgresql_scaler_aws_iam.go

```diff
@@ -0,0 +1,192 @@
+package scalers
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/feature/rds/auth"
+	"github.com/go-logr/logr"
+
+	kedav1alpha1 "github.com/kedacore/keda/v2/apis/keda/v1alpha1"
+)
+
+const (
+	// AWS IAM authentication token validity period (tokens are valid for 15 minutes)
+	awsIAMTokenValidity = 14 * time.Minute // Refresh 1 minute before expiry
```

Comment:
Do you think 1 min is enough , especially in case of network hiccups etc ? How about percentage base where we renew it once it has passed 70-80% of its validity?